### PR TITLE
feat: Create a topbar navigation menu portlet - EXO-60886 -  Meeds-io/MIPs#36 

### PR DESCRIPTION
**Pull request by hakermi**
Create a topbar navigation menu portlet

### CODE REVIEW

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/core-configuration.xml [hunk 1]
```diff
@@ -730,6 +730,7 @@
             <property name="exo.feature.StreamFilter.enabled" value="${exo.feature.StreamFilter.enabled:false}" />
             <property name="exo.feature.SpaceWebNotifications.enabled" value="${exo.feature.StreamFilter.enabled:true}" />
             <property name="exo.feature.publisherRolePromotion.enabled" value="${exo.feature.publisherRolePromotion.enabled:false}" />
+            <property name="exo.feature.navigationPortlet.enabled" value="${exo.feature.navigationPortlet.enabled:false}" />
```
Comment (boubaker) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/core-configuration.xml]: should be deleted, see decision of product committee here https://github.com/Meeds-io/MIPs/issues/36#issuecomment-1375711342

Comment (hakermi) [reply extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/core-configuration.xml]: done

**Comment by boubaker** [comment]
Please check Sonar issues by resolving them or marking them as resolved

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/core-configuration.xml (hunk 1 above)
Comment (boubaker) [reply extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/core-configuration.xml]: ok, thanks

**Comment by hakermi** [comment]
> Please check Sonar issues by resolving them or marking them as resolved

How can i mark them as resolved?

The code smells are fake alerts because the file headers already added.
The security hotspots are also fake alerts

**Comment by boubaker** [comment]
Could you please add commits on feature/mips as well to complete the review in functional & technical perspective and add missing configuration on `meeds` site.

**Comment by hakermi** [comment]
> Could you please add commits on feature/mips as well to complete the review in functional & technical perspective and add missing configuration on `meeds` site.

done
https://github.com/Meeds-io/social/pull/2137
https://github.com/Meeds-io/gatein-portal/pull/515
https://github.com/Meeds-io/platform-ui/pull/583

**Review by boubaker — APPROVED** [review]

**Pull request merged**